config/database.yml

```diff
@@ -4,7 +4,16 @@
 # https://github.com/rails/rails/issues/38924#issuecomment-612881746
 # i have to unroll some parts so that it's simple enough for rails to figure out that I want to use multi-db
 
+global_pg_db_config: &global_pg_db_config
+  variables:
+    # our dbs use SSDs so random access is almost as cheap as seq_page_cost
+    random_page_cost: 1.1
+    # can improve performance for queries that involve large sorts or hash operations but increasing this value too
+    # much could lead to excessive memory usage
+    work_mem: 8MB
```

Comment:
Let's check with @yakloinsteak on this one.  I think it has the potential to massively increase memory usage depending on number of connections allowed.

Reply:
Agreed, I also don't have a clear indication that this is a problem; the relevant monitoring (Temp bytes / files) isn't available to me. But if I had to guess, I'd say the default of 4MB is likely to be too low

Reply:
The first value sounds like a really good idea to test. Might be worth having a little more explanation and possibly set seq_page_cost as well (even if you're just setting it to the default). Mostly to help out those that aren't familiar with the settings. This could be a huge win. I always presumed that RDS set sane values assuming SSDs, but @ttoomey is spot on. Feels like something we would want across the board in all our postgres databases. Thanks for bringing this up, Theron!

Hard to say about the safety of the second one, but I'm definitely on board with increasing it. It's at the default of 4kB right now, so this is a pretty big jump. I'd probably increase it less on the first round. It's kind of a function of the number of connections and the amount of RAM available and what kinds of queries are being run. Maybe start with 8kB to double it? Or maybe 250kB.

Reply:
Oh, I misread the numbers in the database. It's 4096kB, not 4kB. 8MB might actually be okay.